### DISCUSSION TITLE
Snapshot RaspberryMatic-3.61.5.20211124-1e1ca3-rpi3 - - - -  TEST

### DISCUSSION THREAD

**Discussion by ghost** [OpenCCU/OpenCCU#1562] [🇩🇪 Fragen&Antworten] [answered]

**Comment by jens-maus** [comment, marked as answer]
?

**Comment by ghost** [comment]


**Reply by jens-maus** [reply]
Bekannt und schon beseitigt: https://github.com/jens-maus/RaspberryMatic/commit/7c47eaeee7d7148ea4c3167cb2e3b4e8433da6b9

Nightly builds sind nightly builds.

**Reply by ghost** [reply]
sorry , I think , I can little help with tests

**Reply by jens-maus** [reply]
Then report this as an issue ticket next time, but make sure it isn't fixed already in the latest commit history.
